etc/builtin.conf

```diff
@@ -97,3 +97,15 @@ sub-shadow-offset=4
 [box]
 profile=osd-box
 profile=sub-box
+
+[image]
```

Comment:
I'd also add `video-aspect-override=0` here because image files are never tagged properly and none of them actually have non-square pixels in most cases even if it claims to do so

Reply:
Done, also replaced osc=no with script-opt=osc-deadzonesize=0.9.

Reply:
Might as well go all the way with deadzonesize=1?

Reply:
Oh I thought 1 was the whole window, but it's the area above the OSC. Done.